Gemfile.lock

```diff
@@ -258,14 +263,14 @@ GEM
     therubyracer (0.11.4)
       libv8 (~> 3.11.8.12)
       ref
-    thin (1.7.0)
+    thin (1.7.2)
       daemons (~> 1.0, >= 1.0.9)
       eventmachine (~> 1.0, >= 1.0.4)
       rack (>= 1, < 3)
     thor (0.20.3)
-    thread_safe (0.3.5)
+    thread_safe (0.3.6)
     tilt (1.4.1)
-    tinymce-rails (4.4.1)
+    tinymce-rails (5.0.5)
```

Comment:
I looked at https://www.tiny.cloud/docs/migration-from-4x/  and didn't see anything that was impacted.